### PR TITLE
Aceptalo Pedro

### PR DESCRIPTION
**Pull request by Robertomendivil97**

### CODE REVIEW

File: hola_pedro.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hola hola
```
Comment (pedrosc97) [inline hola_pedro.txt:1]: Identation mistake

**Review by pedrosc97 — CHANGES_REQUESTED** [review]
Syntax errors

File: hola_pedro.txt (hunk 1 above)
Comment (Robertomendivil97) [reply hola_pedro.txt:1]: ok

**Review by pedrosc97 — APPROVED** [review]

File: hola_pedro.txt (hunk 1 above)
Comment (pedrosc97) [inline hola_pedro.txt:1]: iin

**Review by pedrosc97 — CHANGES_REQUESTED** [review]

**Review by vSebas — APPROVED** [review]
Approved

**Review by pedrosc97 — APPROVED** [review]

**Pull request merged**